<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWindow</class>
 <widget class="QMainWindow" name="InputWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#InputWindow
{
	border-image: url(:/Desktop/pic.png);
	background-color: rgb(255, 255, 255);
	
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="single_input">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow#InputWindow QPushButton#single_input
{
	
	background-color: rgb(156, 217, 255);
  color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

}

QMainWindow#InputWindow QPushButton#single_input:hover
{
	background-color: rgb(130,190, 240);
}</string>
    </property>
    <property name="text">
     <string>单个录入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 24pt &quot;宋体&quot;;</string>
    </property>
    <property name="text">
     <string>请选择成绩录入方式</string>
    </property>
   </widget>
   <widget class="QPushButton" name="excel_input">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QMainWindow#InputWindow QPushButton#excel_input
{
	
	background-color: rgb(156, 217, 255);
  color: rgb(0,0,0);

border:0px groove gray;border-radius:
7px;padding:2px 4px;
font: 18pt &quot;Candara&quot;;

}

QMainWindow#InputWindow QPushButton#excel_input:hover
{
	background-color: rgb(130,190, 240);
}</string>
    </property>
    <property name="text">
     <string>EXCEL导入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_input_return">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>450</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QMainWindow#InputWindow QPushButton#btn_input_return
{
   font: 18pt &quot;宋体&quot;;
	
	background-color: rgb(255, 255, 255);
color: rgb(0,0,0);
}

QMainWindow#InputWindow QPushButton#btn_input_return:hover
{
	
	background-color: rgb(195, 195, 195);
}</string>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
